sw/device/silicon_creator/mask_rom/mask_rom.c

```diff
@@ -30,6 +30,11 @@
 
 #include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
 
+enum {
+  kMaskRomExpectedCheckCount1 = 1,
+  KMaskRomExpectedCheckCount3 = 3,
```

Comment:
nit: Maybe we should either use more descriptive names, e.g. `kMaskRomExpectedCheckCountInit`, `kMaskRomExpectedCheckCountTryBoot`, or use `sec_mmio_check_counters(/*expected_check_count=*/3)`.